### ISSUE TITLE
Errors in translation to SPARQL

### ISSUE THREAD

**Issue by cmungall** [balhoff/dosdp-tools#15]
From the GO DP repo work (https://github.com/geneontology/go-ontology/issues/13472):

```
$ make validated_patterns/import_across_membrane.tsv
dosdp-tools --prefixes=prefixes.yaml --ontology=../ontology/go-edit.obo --reasoner=elk --obo-prefixes=true --template=validated_patterns/import_across_membrane.yaml --outfile=validated_patterns/import_across_membrane.tsv query
15:35:55.403 [main] DEBUG Jena - Jena initialization
15:35:55.528 [main] DEBUG org.apache.jena.util.FileManager - Add location: LocatorFile
15:35:55.528 [main] DEBUG org.apache.jena.util.FileManager - Add location: ClassLoaderLocator
15:35:55.530 [main] DEBUG org.apache.jena.util.LocationMapper - Failed to find configuration: file:location-mapping.rdf;file:location-mapping.n3;file:location-mapping.ttl;file:etc/location-mapping.rdf;file:etc/location-mapping.n3;file:etc/location-mapping.ttl
15:35:55.530 [main] DEBUG org.apache.jena.util.FileManager - Add location: LocatorFile
15:35:55.530 [main] DEBUG org.apache.jena.util.FileManager - Add location: LocatorURL
15:35:55.530 [main] DEBUG org.apache.jena.util.FileManager - Add location: ClassLoaderLocator
15:35:55.535 [main] DEBUG org.apache.jena.riot.system.stream.JenaIOEnvironment - Failed to find configuration: location-mapping.ttl;location-mapping.rdf;location-mapping.n3;etc/location-mapping.rdf;etc/location-mapping.n3;etc/location-mapping.ttl
Exception in thread "main" java.lang.ExceptionInInitializerError
        at org.apache.jena.rdf.model.impl.ResourceImpl.fresh(ResourceImpl.java:135)
        at org.apache.jena.rdf.model.impl.ResourceImpl.<init>(ResourceImpl.java:85)
        at org.apache.jena.rdf.model.ResourceFactory$Impl.createResource(ResourceFactory.java:297)
        at org.apache.jena.rdf.model.ResourceFactory.createResource(ResourceFactory.java:92)
        at org.apache.jena.vocabulary.RDFS.resource(RDFS.java:35)
        at org.apache.jena.vocabulary.RDFS.<clinit>(RDFS.java:40)
        at org.apache.jena.vocabulary.RDFS$Nodes.<clinit>(RDFS.java:66)
        at org.phenoscape.owlet.Owlet$.<init>(Owlet.scala:171)
        at org.phenoscape.owlet.Owlet$.<clinit>(Owlet.scala)
        at org.phenoscape.owlet.OwletManchesterSyntaxDataType$SerializableClassExpression$.asOMN$extension(OwletManchesterSyntaxDataType.scala:33)
        at org.monarchinitiative.dosdp.SPARQL$$anonfun$3.apply(SPARQL.scala:63)
        at org.monarchinitiative.dosdp.SPARQL$$anonfun$3.apply(SPARQL.scala:60)
        at scala.collection.TraversableLike$$anonfun$flatMap$1.apply(TraversableLike.scala:241)
        at scala.collection.TraversableLike$$anonfun$flatMap$1.apply(TraversableLike.scala:241)
        at scala.collection.mutable.ResizableArray$class.foreach(ResizableArray.scala:59)
        at scala.collection.mutable.ArrayBuffer.foreach(ArrayBuffer.scala:48)
        at scala.collection.TraversableLike$class.flatMap(TraversableLike.scala:241)
        at scala.collection.AbstractTraversable.flatMap(Traversable.scala:104)
        at org.monarchinitiative.dosdp.SPARQL$.triplesFor(SPARQL.scala:60)
        at org.monarchinitiative.dosdp.SPARQL$.queryFor(SPARQL.scala:33)
        at org.monarchinitiative.dosdp.cli.Query$.run(Query.scala:36)
        at org.monarchinitiative.dosdp.cli.Main$$anonfun$1.apply(Main.scala:30)
        at org.monarchinitiative.dosdp.cli.Main$$anonfun$1.apply(Main.scala:30)
        at scala.Option.foreach(Option.scala:257)
        at org.monarchinitiative.dosdp.cli.Main$.delayedEndpoint$org$monarchinitiative$dosdp$cli$Main$1(Main.scala:30)
        at org.monarchinitiative.dosdp.cli.Main$delayedInit$body.apply(Main.scala:28)
        at scala.Function0$class.apply$mcV$sp(Function0.scala:34)
        at scala.runtime.AbstractFunction0.apply$mcV$sp(AbstractFunction0.scala:12)
        at scala.App$$anonfun$main$1.apply(App.scala:76)
        at scala.App$$anonfun$main$1.apply(App.scala:76)
        at scala.collection.immutable.List.foreach(List.scala:381)
        at scala.collection.generic.TraversableForwarder$class.foreach(TraversableForwarder.scala:35)
        at scala.App$class.main(App.scala:76)
        at org.monarchinitiative.dosdp.cli.Main$.main(Main.scala:28)
        at org.monarchinitiative.dosdp.cli.Main.main(Main.scala)
Caused by: java.lang.NullPointerException
        at org.apache.jena.rdf.model.impl.ModelCom.add(ModelCom.java:1191)
        at org.apache.jena.assembler.assemblers.AssemblerGroup$ExpandingAssemblerGroup.implementWith(AssemblerGroup.java:92)
        at org.apache.jena.assembler.ConstAssembler.general(ConstAssembler.java:137)
        at org.apache.jena.sparql.core.assembler.AssemblerUtils.registerDataset(AssemblerUtils.java:68)
        at org.apache.jena.sparql.core.assembler.AssemblerUtils.init(AssemblerUtils.java:60)
        at org.apache.jena.query.ARQ.init(ARQ.java:590)
        at org.apache.jena.sparql.system.InitARQ.start(InitARQ.java:29)
        at org.apache.jena.system.JenaSystem.lambda$init$50(JenaSystem.java:119)
        at java.util.ArrayList.forEach(ArrayList.java:1249)
        at org.apache.jena.system.JenaSystem.forEach(JenaSystem.java:194)
        at org.apache.jena.system.JenaSystem.forEach(JenaSystem.java:171)
        at org.apache.jena.system.JenaSystem.init(JenaSystem.java:117)
        at org.apache.jena.graph.NodeFactory.<clinit>(NodeFactory.java:32)
        ... 35 more
make: *** [validated_patterns/import_across_membrane.yaml] Error 1
```

**Comment by balhoff** [comment]
Should be fixed by ad22841fd0d2170a48358465f7efb7e8f8d8a646.

**Comment by balhoff** [comment]
I'll make a release.
